docs/docs/guides/overview.md

```diff
@@ -0,0 +1,10 @@
+# Using DBSP
+
+DBSP can be used in two ways:
+
+* You can use the Rust DBSP implementation and write directly Rust
+  programs that perform stream processing
+
+* You can write SQL programs and use our SQL compiler to generate
```

Comment:
we should add guides aimed at end-users. writing rust dbsp programs or invoking the compiler directly is for advanced users or dbsp developers and hopefully not needed by normal users. Instead I think the guides should teach these two things:

- How to create and run a dbsp pipeline with the python API
- How to create and run a dbsp pipeline with the UI

we can keep your guides too but if they won't be used by 90% of the users maybe it's better to put them under a separate category e.g., under Contribute to DBSP where we can teach those things?

Reply:
At this point I don't know how to write a guide for users, so I write a guide for developers.
All are useful.